peachjam/templates/peachjam/layouts/base.html

```diff
@@ -62,7 +62,8 @@
     {% block head-css %}{% endblock %}
     {% block head-js %}{% endblock %}
   </head>
-  <body class="{% block body-class %}{% endblock %}">
+  <body hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}'
+        class="{% block body-class %}{% endblock %}">
```

Comment:
@Sandravaphilips is this necessary? if the csrf token is in the forms we post it should work, or it can be added to the button that makes the request. keeping the behaviour local